docs/markdown/flexible-plots.md

```diff
@@ -1,31 +1,53 @@
 ## Flexible plots
 
-By default, XYPlot requires a width and a height. There are times, however, when you'd like your chart to take all the space it can. For these cases, React-vis provides a helper function to make one or both dimensions of your chart flexible.
+By default, XYPlot requires a width and a height. There are times, however, when you'd like your chart to take all the space it can.
+For these cases, React-vis provides three different types of `XYPlot` with different flexible dimensions:
+
+  - FlexibleWidthXYPlot
+  - FlexibleHeightXYPlot
+  - FlexibleXYPlot
+
+and the associated helper functions that have been used to create these flexible components.
```

Comment:
maybe 

as well as the associated helper functions (`makeHeightFlexible/makeWidthFlexible/makeVisFlexible`) that have been used to create these flexible components.